### PR TITLE
Trying something with Kenjiro's PR

### PR DESCRIPTION
**Pull request by markusthoemmes**
As per title, some investigation.

### CODE REVIEW

**Comment by markusthoemmes** [comment]
/retest

Finalizer hang?

**Comment by markusthoemmes** [comment]
/test 4.3-upgrade-tests-aws-ocp-43

Setup issue.

**Comment by markusthoemmes** [comment]
/test 4.3-upgrade-tests-aws-ocp-43

Setup issue.

**Comment by markusthoemmes** [comment]
/retest

Setup. Thanks CI-operator.

**Comment by markusthoemmes** [comment]
/retest

Setup setup setup.

**Comment by markusthoemmes** [comment]
/retest

Setup setup setup.

File: knative-operator/pkg/webhook/knativeserving/webhook_mutating.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func MutatingWebhook(mgr manager.Manager) (webhook.Webhook, error) {
 	return builder.NewWebhookBuilder().
 		Name("mutating.knativeserving.openshift.io").
 		Mutating().
-		Operations(admissionregistrationv1beta1.Create).
+		Operations(admissionregistrationv1beta1.Create, admissionregistrationv1beta1.Update).
```
Comment (jcrossley3) [inline knative-operator/pkg/webhook/knativeserving/webhook_mutating.go]: Are you sure this is needed? I would expect the controller to handle this.

Comment (markusthoemmes) [reply knative-operator/pkg/webhook/knativeserving/webhook_mutating.go]: And the controller even does, but it leads to weird behavior when you override the entirety of KnativeServing with an update. The result you get back is not what's reflected on the cluster.

**Comment by markusthoemmes** [comment]
/retest

Setup issues, what else?!

**Comment by nak3** [comment]
/retest

setup issue again..

**Comment by nak3** [comment]
LGTM! Thanks @markusthoemmes !

**Comment by markusthoemmes** [comment]
outdated.

**Pull request closed without merging**